fix(override-extend): update override-extend-variable module content (#211)
patternfly/training-scenarios: html-css/override-extend-variables/step3-instructions.md

Reviewer: sb: Now that you have practiced overriding variables, practice extending PatternFly's system to create a new element in the card. You will add a separator element to the card.

> ## Extend a PatternFly component to achieve a new design

Reviewer 2: Remove this line we don't use these anymore

> Now that you have practiced overriding variables, you will practice extending PatternFly's system to create a new element in the card. You will add a separator element to the card.

Reviewer 2: Remove instance of "you"

 
1) <strong>Add the html markup.</strong> 
In `index.html` add the html markup for the separator element that is a sibling to the `pf-c-card__body` element. Add this new code between `pf-c-card__header` and `pf-c-card__body` and `pf-c-card__body` and `pf-c-card__footer`. You will prefix the variable name with `my-app` so that it can be easily distinguished from the PatternFly codebase.
 
```
<div class="my-app-c-card__separator">
</div>
```
 
2) <strong>Add a variable for the height.</strong>
In the `myapp.scss` file, you will add new styles for `my-app-c-card__separator`. The first variable to add is for height and it should use PatternFly's global variable for small border width, and go under the variable declarations inside the `.pf-c-card` block.
 
```
  --my-app-c-card__separator--Height: var(--pf-global--BorderWidth--sm);
```
 

> 3) <strong>Add a variable for background color.</strong> The second variable to add is for background color and it should use PatternFly's global variable for border color.

Reviewer 2: Return statement after the first sentence

 
```
  --my-app-c-card__separator--BackgroundColor: var(--pf-global--BorderColor--100);
```
 

> 4) <strong>Add a variable for the margin.</strong> The third variable to add is for the margin and it should use PatternFly's global variable for spacing.

Reviewer 2: Return statement after the first sentence

> Overriding variables have been practiced. Now practice extending PatternFly's system to create a new element in the card. Add the separator element to the card.

Reviewer 3: ```suggestion
Now practice extending PatternFly's system to create a new element in the card. Add the separator element to the card.
```

> Now that you have practiced overriding variables, you will practice extending PatternFly's system to create a new element in the card. You will add a separator element to the card.

Reviewer: ```suggestion
Overriding variables have been practiced. Now practice extending PatternFly's system to create a new element in the card. Add the separator element to the card.
```

 
1) <strong>Add the html markup.</strong> 

> In `index.html` add the html markup for the separator element that is a sibling to the `pf-c-card__body` element. Add this new code between `pf-c-card__header` and `pf-c-card__body` and `pf-c-card__body` and `pf-c-card__footer`. You will prefix the variable name with `my-app` so that it can be easily distinguished from the PatternFly codebase.

Reviewer: ```suggestion
In `index.html` add the html markup for the separator element that is a sibling to the `pf-c-card__body` element. Add this new code between `pf-c-card__header` and `pf-c-card__body` and `pf-c-card__body` and `pf-c-card__footer`. Then prefix the variable name with `my-app` so that it can be easily distinguished from the PatternFly codebase.
```

 
```
<div class="my-app-c-card__separator">
</div>
```
 
2) <strong>Add a variable for the height.</strong>

> In the `myapp.scss` file, add new styles for `my-app-c-card__separator`. The first variable to add is for height and it should use PatternFly's global variable for small border width, and go under the variable declarations inside the `.pf-c-card` block.

Reviewer 3: ```suggestion
In the `myapp.scss` file, add new styles for `my-app-c-card__separator`. First, add a variable for height. It should use PatternFly's global variable for small border width. Add it under the variable declarations inside the `.pf-c-card` block.
```

> In the `myapp.scss` file, you will add new styles for `my-app-c-card__separator`. The first variable to add is for height and it should use PatternFly's global variable for small border width, and go under the variable declarations inside the `.pf-c-card` block.

Reviewer: ```suggestion
In the `myapp.scss` file, you add new styles for `my-app-c-card__separator`. The first variable to add is for height and it should use PatternFly's global variable for small border width, and go under the variable declarations inside the `.pf-c-card` block.
```

> In the `myapp.scss` file, you will add new styles for `my-app-c-card__separator`. The first variable to add is for height and it should use PatternFly's global variable for small border width, and go under the variable declarations inside the `.pf-c-card` block.

Reviewer: ```suggestion
In the `myapp.scss` file,  add new styles for `my-app-c-card__separator`. The first variable to add is for height and it should use PatternFly's global variable for small border width, and go under the variable declarations inside the `.pf-c-card` block.
```

 
```

>   --my-app-c-card__separator--Height: var(--pf-global--BorderWidth--sm);

Reviewer 2: ```suggestion
--my-app-c-card__separator--Height: var(--pf-global--BorderWidth--sm);
```

```
 
3) <strong>Add a variable for background color.</strong> 

> The second variable to add is for background color and it should use PatternFly's global variable for border color.

Reviewer 3: ```suggestion
Add a second variable for background color. It should use PatternFly's global variable for border color.
```

 
```
  --my-app-c-card__separator--BackgroundColor: var(--pf-global--BorderColor--100);
```
 
4) <strong>Add a variable for the margin.</strong> 

> The third variable to add is for the margin and it should use PatternFly's global variable for spacing.

Reviewer 3: ```suggestion
Add a third variable for the margin. It should use PatternFly's global variable for spacing.
```
